build.sh

```diff
@@ -31,13 +25,11 @@ srcdir=$(pwd)
 release="30"
 
 configure_yum_repos() {
-    if [ -n "${ISFEDORA}" ]; then
-        # Add continuous tag for latest build tools and mark as required so we
-        # can depend on those latest tools being available in all container
-        # builds.
-        echo -e "[f$release-coreos-continuous]\nenabled=1\nmetadata_expire=1m\nbaseurl=https://kojipkgs.fedoraproject.org/repos-dist/f$release-coreos-continuous/latest/\$basearch/\ngpgcheck=0\nskip_if_unavailable=False\n" > /etc/yum.repos.d/coreos.repo
+    # Add continuous tag for latest build tools and mark as required so we
+    # can depend on those latest tools being available in all container
+    # builds.
+    echo -e "[f$release-coreos-continuous]\nenabled=1\nmetadata_expire=1m\nbaseurl=https://kojipkgs.fedoraproject.org/repos-dist/f$release-coreos-continuous/latest/\$basearch/\ngpgcheck=0\nskip_if_unavailable=False\n" > /etc/yum.repos.d/coreos.repo
```

Comment:
Could replace this with a heredoc to make the \n's a little easier to understand, but that can wait for a different PR if you want.